### PR TITLE
Create amishasao.md

### PR DESCRIPTION
**Pull request by amishasao**
## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

File: amishasao.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+7. attach the nylon covering around the backbone
+8. attach external components as needed
+
+## Budget - as of now :D
+
+| Product            | Supplier/Link           | Cost    |
+| ------------------ | ----------------------- | ------- |
+| Arduino nano       | https://a.co/d/8cjCErK  | $30.99  |
+| 10 Servo motors    | https://a.co/d/aApiF3y  | $19.95  |
+| 6 haptic sensors   | shorturl.at/aprDH       | $12.69  |
```
Comment (garyhtou) [inline amishasao.md:34]: `shorturl.at/aprDH` seem to be a dead link, could you please update it with a working link?

**Review by garyhtou — CHANGES_REQUESTED** [review]
Hey @amishasao, great to see that you're building a robotic glove to help with mobility!

Please see my comment about the link for your haptic sensors.

File: amishasao.md (hunk 1 above)
Comment (amishasao) [reply amishasao.md]: Hi, it should be fixed!

File: amishasao.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+7. attach the nylon covering around the backbone
+8. attach external components as needed
+
+## Budget - as of now :D
+
+| Product            | Supplier/Link           | Cost    |
+| ------------------ | ----------------------- | ------- |
+| Arduino nano       | https://a.co/d/8cjCErK  | $30.99  |
+| 10 Servo motors    | https://a.co/d/aApiF3y  | $19.95  |
+| 6 haptic sensors   | https://www.aliexpress.us/item/2251832669872365.html?spm=a2g0o.ppclist.product.10.1a6eW5W6W5W6Ff&pdp_npi=2%40dis%21USD%21US%20%243.37%21%241.79%21%21%21%21%21%4021032fa016724082409855056ebc66%2165296889214%21btf&_t=pvid%3A43deb1cb-b818-4216-8c3b-dff0952b8e32&afTraceInfo=32856187117__pc__pcBridgePPC__xxxxxx__1672408241&gatewayAdapt=glo2usa&_randl_shipto=US       | $12.69  |
```
Comment (garyhtou) [inline amishasao.md:34]: Will you be needing some wire and/or a soldering kit to connect the sensors to the Arduino?

File: amishasao.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+6. program the arduino to connect the servo motor motion with the pressure sensor live data
+7. attach the nylon covering around the backbone
+8. attach external components as needed
+
+## Budget - as of now :D
+
+| Product            | Supplier/Link           | Cost    |
+| ------------------ | ----------------------- | ------- |
+| Arduino nano       | https://a.co/d/8cjCErK  | $30.99  |
+| 10 Servo motors    | https://a.co/d/aApiF3y  | $19.95  |
```
Comment (garyhtou) [inline amishasao.md:33]: Will the Arduino be able to power the 6 servos? Or does there need to be some external power?

File: amishasao.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+5. attach haptic sensors to sense the pressure live
+6. program the arduino to connect the servo motor motion with the pressure sensor live data
+7. attach the nylon covering around the backbone
+8. attach external components as needed
+
+## Budget - as of now :D
+
+| Product            | Supplier/Link           | Cost    |
+| ------------------ | ----------------------- | ------- |
+| Arduino nano       | https://a.co/d/8cjCErK  | $30.99  |
```
Comment (garyhtou) [inline amishasao.md:32]: Could you please double-check if the nano has enough pins to connect with all the servos and sensors?

**Review by garyhtou — CHANGES_REQUESTED** [review]
Thanks for updating the link @amishasao!

Do you happen to have a diagram or a rough model of the glove? I'm curious as to your plans for arranging and mounting the servo motors to the glove. Would love to hear your thoughts on this.

I've also left a couple of comments!

File: amishasao.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+| 10 Servo motors    | https://a.co/d/aApiF3y  | $19.95  |
+| 6 haptic sensors   | https://www.aliexpress.us/item/2251832669872365.html?spm=a2g0o.ppclist.product.10.1a6eW5W6W5W6Ff&pdp_npi=2%40dis%21USD%21US%20%243.37%21%241.79%21%21%21%21%21%4021032fa016724082409855056ebc66%2165296889214%21btf&_t=pvid%3A43deb1cb-b818-4216-8c3b-dff0952b8e32&afTraceInfo=32856187117__pc__pcBridgePPC__xxxxxx__1672408241&gatewayAdapt=glo2usa&_randl_shipto=US       | $12.69  |
+| 1kg PLA filament   | https://a.co/d/8SBnmid  | $16.99  | (probably won't need but just in case)
+| 1kg Nylon filament | https://a.co/d/91SR21T  | $24.99  |
+| Velcro strips      | https://a.co/d/08rdYgR  | $5.97   |
+| Total              |                         | $111.58 |
```
Comment (garyhtou) [inline amishasao.md:38]: I also wanted to double-check whether these prices include shipping and tax. We'd love to include that in the budget to make sure you have the funds necessary to complete the project!

File: amishasao.md (hunk 3 above)
Comment (amishasao) [reply amishasao.md:33]: Yes, I think so. There does not seem to be a need for any external power.

File: amishasao.md (hunk 2 above)
Comment (amishasao) [reply amishasao.md:34]: Yes, I think so. I'll try to find a suitable one asap!

File: amishasao.md (hunk 4 above)
Comment (amishasao) [reply amishasao.md:32]: Yes, it does have enough pins to connect with all the servos and sensors.

File: amishasao.md (hunk 5 above)
Comment (amishasao) [reply amishasao.md:38]: Some of them don't have shipping and tax included. I can update the file later today to include those!

**Comment by amishasao** [comment]
> Thanks for updating the link @amishasao!
> 
> Do you happen to have a diagram or a rough model of the glove? I'm curious as to your plans for arranging and mounting the servo motors to the glove. Would love to hear your thoughts on this.
> 
> I've also left a couple of comments!

File: amishasao.md (hunk 2 above)
Comment (amishasao) [reply amishasao.md:34]: Yes, let me check which one I need.

**Comment by garyhtou** [comment]
hi @amishasao feel free to give me a ping once you've updated the file! thanks :)

**Comment by amishasao** [comment]
> hi @amishasao feel free to give me a ping once you've updated the file! thanks :)

Hey, I have updated the file! Feel free to take a look when you get a chance. Thanks!

File: amishasao.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+8. attach external components as needed
+
+## Budget - as of now :D
+
+| Product            | Supplier/Link           | Cost    |
+| ------------------ | ----------------------- | ------- |
+| Arduino nano       | https://a.co/d/8cjCErK  | $30.99  |
+| 10 Servo motors    | https://a.co/d/aApiF3y  | $19.95  |
+| 6 haptic sensors   | https://www.aliexpress.us/item/2251832669872365.html?spm=a2g0o.ppclist.product.10.1a6eW5W6W5W6Ff&pdp_npi=2%40dis%21USD%21US%20%243.37%21%241.79%21%21%21%21%21%4021032fa016724082409855056ebc66%2165296889214%21btf&_t=pvid%3A43deb1cb-b818-4216-8c3b-dff0952b8e32&afTraceInfo=32856187117__pc__pcBridgePPC__xxxxxx__1672408241&gatewayAdapt=glo2usa&_randl_shipto=US       | $12.69  |
+| 1kg PLA filament   | https://a.co/d/8SBnmid  | $18.99  | (probably won't need but just in case)
```
Comment (cfanoulis) [inline amishasao.md:35]: hey, can you please confirm if you'll be using this? if not, remove it from your budget - we're trying to keep an accurate bill of materials!

File: amishasao.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+I am going to be building a soft robotic glove for individuals with limited hand motor function to improve functionality faster and be able to grip, hold, and release objects with ease. This could be a potentially helpful tool for stroke and other patients attempting to gain back their motor function via therapy or rehabilitation. The glove itself is going to do two main functions: sense pressure (from the fingertips and wrist) and rotate the appendages (based on the pressure reading). So, for example, if I wanted to pick up a book from my desk, I would get to the book, put pressure on my wrist (which will rotate the glove and, thus, my hand downward), put pressure on my fingertips to try to hold the book, hold the book (given the glove's grip in place), and realease the pressure from my fingertips when wanting to drop the book.
+
+Why are you excited to build it?
+I am excited to build this because I believe that it can have real life implications and become a cost-effective option for patients to regain their hand motor function. Additionally, I am really excited to 3d print nylon and experiment with haptic sensors judging the movement of the wearable glove.
+
+## Plan
+
```
Comment (cfanoulis) [inline amishasao.md:17]: I see you've got quite a strong plan! Can you please include any lnks of guides you'd be following or any inspiration you've had? thanks (its fine if there's none, just let me know!)

**Review by cfanoulis — CHANGES_REQUESTED** [review]
Hey! I'm taking over this PR, please address some q's below :)

File: amishasao.md (hunk 7 above)
Comment (amishasao) [reply amishasao.md:17]: My main inspiration was that I have a skinny wrist, which makes it kinda hard to do things like handstands--so I was intrigued by motor function at first. I then looked into it and found something like my project idea that was being developed by Harvard scientists to improve hand motor function in rehabilitation patients. The core function seemed similar to: https://www.instructables.com/DIY-Robotic-Hand-Controlled-by-a-Glove-and-Arduino/, so I decided to give it a try:)

Here are a couple more links:
- https://youtu.be/cxZJSMguo_g
- https://www.instructables.com/Arduino-Flex-Sensor-Glove/

File: amishasao.md (hunk 6 above)
Comment (amishasao) [reply amishasao.md]: I've removed it from my budget!

**Comment by amishasao** [comment]
> Hey! I'm taking over this PR, please address some q's below :)

Hey, I've updated the file as asked!

**Review by cfanoulis — APPROVED** [review]
omg the background for this project is so heartwarming 🥺 this is an amazing proposal and i cannot even begin to express how happy i am to see this fully fleshed out!

congrats, you got cash! a banker from @hackclub/bank-ops will he in touch shortly to get the $$$ flowin'!

**Review by MelSmith104 — COMMENTED** [review]
hey @amishasao i really love your project and i love the reason you want to build this. the next step is to jump on slack to go through how the grant will be processed. can you please send me your slack id so that i can reach out? thank you!

**Comment by amishasao** [comment]
> hey @amishasao i really love your project and i love the reason you want to build this. the next step is to jump on slack to go through how the grant will be processed. can you please send me your slack id so that i can reach out? thank you!

Hey @MelSmith104, my slack id is "Amisha" (we have a DM together, I believe)

**Comment by MelSmith104** [comment]
@amishasao dm'ed you!

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**